idletiming_conn.go

```diff
@@ -69,11 +68,11 @@ func Conn(conn net.Conn, idleTimeout time.Duration, onIdle func()) *IdleTimingCo
 // IdleTimingConn is a net.Conn that wraps another net.Conn and that times out
 // if idle for more than idleTimeout.
 type IdleTimingConn struct {
-	// Keep them at the top to make sure 64-bit alignment, see
+	// Keep it at the top to make sure 64-bit alignment, see
 	// https://golang.org/pkg/sync/atomic/#pkg-note-BUG
-	readDeadline     int64
-	writeDeadline    int64
 	lastActivityTime int64
+	readDeadline     guardedTime
```

Comment:
We can convert between epoch and zero value when `Set*Deadline`, but that makes the code even harder to read. Consider `idleTimeout` is usually in seconds, the performance gain by using atomic operations is neglectable.